package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "publish-action",
-  "version": "0.2.1",
+  "version": "1.0.0",
```

This action is for internal usage only and we do not update major version for this action. We update only minor version in case of any breaking changes.  
```suggestion
  "version": "0.3.0",
```